src/views/open-api-docs-view/types.ts

```diff
@@ -163,3 +164,39 @@ export interface OpenApiDocsViewProps {
 	 */
 	_placeholder: $TSFixMe
 }
+
+/**
+ * Type definition for OpenApiDocsView server-side page configuration
+ */
+export interface OpenApiDocsPageConfig {
```

Comment:
I've added a type definition for `OpenApiDocsPageConfig` - it felt useful to clarify what each part of the page configuration does (previously these were comments inline in the page file).